cms/djangoapps/contentstore/views/course.py

```diff
@@ -852,6 +852,8 @@ def _create_or_rerun_course(request):
 
     try:
         org = request.json.get('org')
+        enabled_organizations = get_enabled_organizations(request)
```

Comment:
Getting value for  `org` should be outside of `try/catch` block:
```python
org = request.json.get('org')
enabled_organizations = get_enabled_organizations(request)
if enabled_organizations:
    org = enabled_organizations[0].get('short_name', '')

try:
```

Reply:
done